### PR TITLE
Lesson 39

### PR DESCRIPTION
**Pull request by LudmilaZhuleha**

### CODE REVIEW

File: src/components/Signout/Signout.jsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {useNavigate} from "react-router-dom";
+import Icon from "../Icon/Icon";
+import './Signout.css';
+
+const Signout = () => {
+  const navigate = useNavigate();
+  return (
+    <button onClick={()=>{navigate('/')}} type="text" className="signout-btn">
```
Comment (Marinarium) [inline src/components/Signout/Signout.jsx]: maybe it shpuld be a link

File: src/components/App/App.css [hunk 2]
```diff
@@ -14,8 +14,8 @@
...
     width: 40px;
     height: 40px;
-    color: #FFFFFF;
-    transition-property: color;
-    transition-duration: 300ms;
-    transition-timing-function: ease-in-out;
+    /*color: #FFFFFF;*/
+    /*transition-property: color;*/
+    /*transition-duration: 300ms;*/
+    /*transition-timing-function: ease-in-out;*/
```
Comment (Marinarium) [inline src/components/App/App.css]: please remove unused code from comments

**Review by Marinarium — CHANGES_REQUESTED** [review]

File: src/components/App/App.css (hunk 2 above)
Comment (LudmilaZhuleha) [reply src/components/App/App.css]: Done. Hover effect saved.

File: src/components/App/App.js [hunk 3]
```diff
@@ -13,6 +14,8 @@ import Footer from "../Footer/Footer";
 function App() {
   const [value, setValue] = useState("");
   const [AvailableIsOpen, setAvailableIsOpen] = useState(false);
```
Comment (Marinarium) [inline src/components/App/App.js]: availableIsOpen

File: src/components/App/App.js [hunk 4]
```diff
@@ -13,6 +14,8 @@ import Footer from "../Footer/Footer";
...
+  const [openSignOut, setOpenSignOut] = useState(false);
+  const [accountColor, setAccountColor] = useState('#FFFFFF');
```
Comment (Marinarium) [inline src/components/App/App.js]: better to save color in constants

File: src/components/App/App.js [hunk 5]
```diff
@@ -22,15 +25,21 @@ function App() {
     e.preventDefault();
```
Comment (Marinarium) [inline src/components/App/App.js:26]: e.preventDefault();?is it possible to change button type?

File: src/components/App/App.js [hunk 6]
```diff
@@ -22,15 +25,21 @@ function App() {
...
     setAvailableIsOpen(true);
   };
+
+  const handleSignout = () =>{
+    setOpenSignOut(true);
+    setAccountColor('#F5BD41');
```
Comment (Marinarium) [inline src/components/App/App.js]: please add this color into constants

File: src/components/App/App.js (hunk 4 above)
Comment (LudmilaZhuleha) [reply src/components/App/App.js]: Done

File: src/components/App/App.js (hunk 3 above)
Comment (LudmilaZhuleha) [reply src/components/App/App.js]: Done

File: src/components/App/App.js (hunk 5 above)
Comment (LudmilaZhuleha) [reply src/components/App/App.js:26]: Added type="text" to button, but it did not help. Left as it was with e.preventDefault().

File: src/components/App/App.js (hunk 6 above)
Comment (LudmilaZhuleha) [reply src/components/App/App.js]: Done

**Pull request merged**